programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+
+    var p_num = 0;
+    var y_num = 0;
+    
+    for(let i=0; i<s.length; i++){
+        if ( s[i] === 'p' || s[i] === 'P' ){
+            p_num += 1;
+        } else if( s[i] === 'y' || s[i] === 'Y' )
+            y_num += 1;
+    }
+    if(p_num === y_num){
+        return true;
+    } else {
+        return false;
```

Comment:
```suggestion
    if(p_num === y_num){
        return true;
   
    return false;
```
if문 안에서 return문을 사용하게 되면 밖에서 else를 쓰지 않아도
반대되는 상황에 밖에 있는 return문을 사용하게 됩니다

Reply:
return true와 return false의 들여쓰기 차이는 주의해야하는 부분일까요 ??

Reply:
아 들여쓰기도 중요합니다!